drivers/mspi/Kconfig.nrfe

```diff
@@ -11,6 +11,7 @@ menuconfig MSPI_NRFE
 	select MBOX
 	select IPC_SERVICE
 	select IPC_SERVICE_BACKEND_ICMSG
+	select COUNTER
```

Comment:
Could enabling the timeout be optional? Then, only that config would select counter.